packages/core/src/util/clientInstrumentationCheck.js

```diff
@@ -0,0 +1,33 @@
+/*
+ * (c) Copyright IBM Corp. 2024
+ */
+
+'use strict';
+
+exports.tracerInstrumentationInfo = function tracerInstrumentationInfo() {
+    const instrumentationMap = {
+      usingExperimentalLoaderFlag: '--experimental-loader flag',
+      usingImport: '--import flag',
+      usingRequire: '--require flag',
+      noFlags: 'no additional flags'
+    };
+
+    const usingExperimentalLoaderFlag =
```

Comment:
Qs: Why isn't there a check for `@instana/collector` included here? I noticed that you added checks for other flags.

Reply:
Thanks for catching. That was not added in the checklist.
I will update this and add to checklist